schemas/WebpackOptions.json

```diff
@@ -1408,7 +1408,7 @@
         },
         "strictExportPresence": {
           "description": "Emit errors instead of warnings when imported names don't exist in imported module.",
-          "type": "boolean"
+          "enum": [false, true, "error", "warn", "ignore"]
         },
```

Comment:
The option naming no longer make sense with this change.

Maybe this makes more sense:

``` ts
strictExportPresence: boolean, // no longer recommended
exportsPresence: "warn" | "error" | "auto" | false, // defaults to strictExportPresence ? "error" : "auto"
reexportExportsPresence: "warn" | "error" | "auto" | false, // defaults to exportsPresence
```

it would also allow to disable only `reexportExportsPresence: false` for typescript problems and keep the check on imports.

---

Alternative:

``` ts
strictExportPresence: boolean, // no longer recommended
exportsPresence:
  | "warn" 
  | "error"
  | "auto"
  | "warn-only-import"
  | "error-only-import"
  | "only-import"
  | false,
  // defaults to strictExportPresence ? "error" : "auto"
```

Reply:
I definitely agree that the name is confusing, but couldnt think of a great way to create a new variable which remains backward compatible. Let me give it another go. I like the suggestions you gave above